CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 # go-ipfs changelog
 
-### 0.4.10 - 2017-06-18
+### 0.4.10 - 2017-06-27
+
+Ipfs 0.4.10 is a patch release that contains several exciting new features,
+bugfixes and general improvements.
```

Comment:
Could use one more sentence on why they are exciting for ipfs users. Easier to maintain and recover from processes?